connector/exceptionsconnector/config.go

```diff
@@ -15,6 +15,10 @@ type Dimension struct {
 	Default *string `mapstructure:"default"`
 }
 
+type ExemplarsConfig struct {
+	Enabled bool `mapstructure:"enabled"`
```

Comment:
Do you need to differentiate between unset and false? If so, it would be better to make it a pointer.

Reply:
$0.02 - probably not needed. bool should be fine. unset should be same as default (== disabled)

Reply:
it's gonna make the if statement longer, not sure if it's worth iMO